server_integration_test.go

```diff
@@ -567,6 +604,41 @@ func TestServerGet(t *testing.T) {
 	}
 }
 
+func TestServerRelativeGet(t *testing.T) {
+	tmpWorkingDir, err := ioutil.TempDir("", "sftp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpWorkingDir)
+
+	listenerGo, hostGo, portGo := testServer(t, GOLANG_SFTP, READONLY, tmpWorkingDir)
+	defer listenerGo.Close()
+
+	fn := randName()
+	tmpFileLocal := filepath.Join("/tmp", fn)
+	defer os.RemoveAll(tmpFileLocal)
+
+	t.Logf("get: local %v remote %v ", tmpFileLocal, fn)
+
+	// create a file with random contents. This will be the remote file pulled from the server
+	tmpFileRemoteData := randData(10 * 1024 * 1024)
```

Comment:
The test is about the behavior of a relative get where the base `fn` is used as the [source](https://github.com/pkg/sftp/pull/68/files#diff-3e4bb7147866350f6ffeea6564dfe0e9R630) of the transfer.

The generation of silly goo is, again, simply propagating the patterns that exist in the current tests.